### PR TITLE
Our Company (aka Info Techno) and Hire Us responsive design

### PR DESCRIPTION
**Pull request by pigma15**
Responisve design for 'Our Company' and 'Hire Us'.
Might need some minor tweaks for 'Join Us Now' button after PR with button.css will be implemented.

### CODE REVIEW

File: css/components/hire-us.css [hunk 1]
```diff
@@ -87,54 +99,104 @@
...
+
+@media (max-width: 680px) {
+    .hireus {
+        margin-top: 25px;
+    }
+    .linklist {
+        grid-template-columns: 1fr;
+    }
+    .hireus-container {
+        width: 60%;
```
Comment (bemze) [inline css/components/hire-us.css]: Daryciau 
.hireus-container {
        width: 100%, kad arciau original butu
ir margin sumazinciau iki minimumo

Comment (pigma15) [reply css/components/hire-us.css]: pakeciau width iki 68% nuo 60% ir atitinkamai sumazinau margin, 100% mano akimis ten labai perdaug

**Review by martynas3113 — APPROVED** [review]
LGTM

**Review by bemze — APPROVED** [review]
Mobilios versijos responsive nesueina, bet problema grupei zinoma

**Pull request merged**